lxd/cgroup/cgroup_cpu.go

```diff
@@ -10,10 +10,10 @@ import (
 var DeviceSchedRebalance = make(chan []string, 2)
 
 // TaskSchedulerTrigger triggers a CPU rebalance.
-func TaskSchedulerTrigger(srcType string, srcName string, srcStatus string) {
+func TaskSchedulerTrigger(srcType string, srcName string, srcStatus string, srcPinStrategy string) {
```

Comment:
For clarity can we update the srcType field to be of type instancetype.Type, e.g.

```
func TaskSchedulerTrigger(srcType instancetype.Type, srcName string, srcStatus string) {
```

Then use `srcType.String()` as the value passed to the event channel.

Reply:
Then use `d.dbType` in the call sites.